allennlp/models/simple_tagger.py

```diff
@@ -42,10 +42,10 @@ def __init__(self, vocab: Vocabulary,
         self.tag_projection_layer = TimeDistributed(Linear(self.stacked_encoder.get_output_dim(),
                                                            self.num_classes))
 
-    # pylint: disable=arguments-differ
     def forward(self,  # type: ignore
```

Comment:
We give models which can be `TextFieldEmbedders` a `get_embedding_representations()` method, which calls forward and contains the unpacking logic for what is needed for the representation.